packages/checkout/shipping/shipping-rates-control/shipping-rates-control.js

```diff
@@ -50,14 +50,7 @@ const ShippingRatesControl = ( {
 		}
```

Comment:
The text "Select shipping options below." that is shown when multiple packages are available is no longer visible when we don't have initial shipping, but that's I'm okay giving up.

The other thing is that when you have single item, you're provided two shipping options, initial one and future one, feels a bit redundant and thinking if we can skip it?

Reply:
Does subs core skip it?

Reply:
Yes.